roles/user_models.py

```diff
@@ -0,0 +1,29 @@
+"""
+User data models (mostly used for the Rest Framework)
+"""
+from __future__ import unicode_literals
+
+
+class UserGroup(object):
+    """
+    Class defining the relationship between an user name and a basic group type
+    """
+    def __init__(self, username, group_type):
+        self.username = username
+        self.group_type = group_type
+
+    def __repr__(self):
+        return "<username:{username} | group_type:{group_type}>".format(
+            username=self.username,
+            group_type=self.group_type
+        )
+
+    def __eq__(self, other):
+        try:
```

Comment:
Is it possible to just compare type and `__dict__` here? It would avoid the need for catching an exception and would adapt better to changes in the object